### PR TITLE
add homework 1

### PR DESCRIPTION
**Pull request by lesiaturuta**

### CODE REVIEW

File: homeworks/lesia.turua_lesiaturuta/PokemonProject/style.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  max-width: 575px;
+}
+
+.description {
+  font-size: 48px;
+  font-weight: 700;
+}
+
+.text {
+  font-size: 22px;
```
Comment (vladyslav-yermolin-moc) [inline homeworks/lesia.turua_lesiaturuta/PokemonProject/style.css:21]: please add a line-heigh from the desing
image

File: homeworks/lesia.turua_lesiaturuta/PokemonProject/style.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
+* {
+  margin: 0;
+  padding: 0;
```
Comment (vladyslav-yermolin-moc) [inline homeworks/lesia.turua_lesiaturuta/PokemonProject/style.css:3]: Also would be great to add some `margin-bottom` to the paragraphs if you reset all default indentation. Cause now they all are stacked together.
image

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

File: homeworks/lesia.turua_lesiaturuta/PokemonProject/style.css (hunk 2 above)
Comment (lesiaturuta) [reply homeworks/lesia.turua_lesiaturuta/PokemonProject/style.css:3]: fix: add margin-top for text

File: homeworks/lesia.turua_lesiaturuta/PokemonProject/style.css (hunk 1 above)
Comment (lesiaturuta) [reply homeworks/lesia.turua_lesiaturuta/PokemonProject/style.css:22]: fix: add line-height for description and text

**Review by vladyslav-yermolin-moc — APPROVED** [review]

**Pull request merged**